### PR TITLE
[APM][.NET] Custom instrumentation requires automatic instrumentation

### PR DESCRIPTION
**Pull request by pierotibou**
### What does this PR do? What is the motivation?
Undocument the possibility to have custom instrumentation without automatic instrumentation in .NET. Indeed, we are dropping this feature in our next major release and would like to avoid new customers to use it.

### Merge instructions

- [X ] Please merge after reviewing

### CODE REVIEW

**Review by andrewlock — APPROVED** [review]

File: content/en/tracing/trace_collection/custom_instrumentation/dotnet.md [hunk 1]
```diff
@@ -25,22 +25,22 @@ further_reading:
...
 This page details common use cases for adding and customizing observability with Datadog APM. For a list of supported runtimes, see the [.NET Framework Compatibility Requirements][1] or the [.NET Core Compatibility Requirements][2].
 
 There are several ways to get more than the [default automatic instrumentation][3]:
 
-- [Through configuration](#instrument-methods-through-configuration), which depends on automatic instrumentation and does not allow you to add specific tags.
-- [Using attributes](#instrument-methods-through-attributes), which depends on automatic instrumentation and allows you to customize operation and resource names.
-- [Using custom code](#custom-instrumentation-with-code), which can work alongside automatic instrumentation or without it. It gives you the most control on the spans.
+- [Through configuration](#instrument-methods-through-configuration), which does not allow you to add specific tags.
+- [Using attributes](#instrument-methods-through-attributes), which allows you to customize operation and resource names.
+- [Using custom code](#custom-instrumentation-with-code). It gives you the most control on the spans.
```
Comment (bouwkast) [inline content/en/tracing/trace_collection/custom_instrumentation/dotnet.md:38]: Nitpicky to follow the style of the other two:

```suggestion
- [Using custom code](#custom-instrumentation-with-code), which gives you the most control on the spans.
```

File: content/en/tracing/trace_collection/custom_instrumentation/dotnet.md [hunk 2]
```diff
@@ -25,22 +25,22 @@ further_reading:
...
 
-You can combine these solutions with each other to achieve the instrumentation detail you want.
+You can combine these solutions with each other to achieve the instrumentation detail you want. All those solutions require automatic instrumentation to be setup first.
```
Comment (bouwkast) [inline content/en/tracing/trace_collection/custom_instrumentation/dotnet.md:40]: Thoughts?

```suggestion
You can combine these solutions with each other to achieve the instrumentation detail you want. However, automatic instrumentation must be setup first.
```

File: content/en/tracing/trace_collection/custom_instrumentation/dotnet.md [hunk 3]
```diff
@@ -25,22 +25,22 @@ further_reading:
       text: '.NET code samples'
 ---
 
+<div class="alert alert-info">
+If you have not yet read the instructions for auto-instrumentation and setup, start with the <a href="https://docs.datadoghq.com/tracing/setup/dotnet-core/">.NET/.NET Core</a> or <a href="https://docs.datadoghq.com/tracing/setup/dotnet-framework/">.NET Framework</a> Setup Instructions.
```
Comment (bouwkast) [inline content/en/tracing/trace_collection/custom_instrumentation/dotnet.md:29]: Super nitpicky to change `auto-instrumentation` to `automatic instrumentation` as it is through the rest of the doc.

```suggestion
If you have not yet read the instructions for automatic instrumentation and setup, start with the <a href="https://docs.datadoghq.com/tracing/setup/dotnet-core/">.NET/.NET Core</a> or <a href="https://docs.datadoghq.com/tracing/setup/dotnet-framework/">.NET Framework</a> Setup Instructions.
```

**Review by bouwkast — APPROVED** [review]
Just some nitpicky stuff on words :)

**Review by zacharycmontoya — APPROVED** [review]

**Review by aliciascott — APPROVED** [review]
Approved from docs side, pending the suggestions from @bouwkast :)

**Comment by pierotibou** [comment]
Thanks @bouwkast. @aliciascott , you can merge now then. Thanks everyone for the reviews

**Pull request merged**